### ISSUE TITLE
Check and update copyright headers

### ISSUE THREAD

**Issue by fweik** [espressomd/espresso#1421]

**Comment by hmenke** [comment]
Please convert this
```cpp
/*
  Copyright (C) 2010,2011,2012,2013,2014,2015,2016 The ESPResSo project
  Copyright (C) 2002,2003,2004,2005,2006,2007,2008,2009,2010
    Max-Planck-Institute for Polymer Research, Theory Group
```
into this
```cpp
/*
  Copyright (C) 2010-2017 The ESPResSo project
  Copyright (C) 2002-2010 Max-Planck-Institute for Polymer Research, Theory Group
```

Ideally one could do something like
```cpp
/* License GPLv3 <http://www.gnu.org/licenses/> */
```

Copyright headers only need to be updated when the file is edited.  If the file was last touched in 2016, then the header must not extend the copyright to 2017.

This is probably worth reading in that regard: https://softwareengineering.stackexchange.com/questions/125836/do-you-have-to-include-a-license-notice-with-every-source-file

**Comment by mkuron** [comment]
The license notice we currently have is common practice in GPLv3 projects (so that the license and warranty disclaimer remain intact even when a file is moved out of the project), while the copyright notice is not. Under German law, you do not need to explicitly assert your copyright to have copyright, so we could just delete the notice entirely without changing anything.
Also, "The ESPResSo project" is not a legal entity, so that line is kind of pointless anyway. The true copyright owners are the individual contributors (and usually their institutions have some kind of implied license to it). Since you can't transfer copyright under German law, the MPIP is probably not the true copyright holder before 2010 either.

**Comment by KaiSzuttor** [comment]
What is the decision here?

**Comment by pdebuyl** [comment]
Some Python projects have something along the lines of

```
/*
  Copyright (C) 2002-2017 The ESPResSo project
```

or

```
/*
  Copyright (C) 2002-2017 The ESPResSo developers
```

with the precise meaning in a file `AUTHORS` (that espresso has already anyway) or in the changelog (numpy for instance).

I don't know the intent here, or the importance of keeping explicitly the MPI Polymer name.

**Comment by fweik** [comment]
I suppose this was fixed by #2198?

**Comment by mkuron** [comment]
Yes, please close this ticket.
